packages/main/src/FileUploader.ts

```diff
@@ -156,6 +181,15 @@ class FileUploader extends UI5Element implements IFormInputElement {
 	@property()
 	value = "";
 
+	/**
+	 * Defines the maximum file size in megabytes which prevents the upload if at least one file exceeds it.
+	 * @default undefined
+	 * @since 2.2.0
```

Comment:
@since 2.3.0 already?

Reply:
i got confused with the little and big versions, reverted back to 2.2 :)